### PR TITLE
Update document for self-host maps server

### PR DESCRIPTION
**Pull request by junqiu-lei**
Signed-off-by: Junqiu Lei <junqiu@amazon.com>

### Description
Update document for self-host maps server followed by https://opensearch.org/docs/latest/dashboards/selfhost-maps-server/ 
### Issues Resolved
https://github.com/opensearch-project/maps/issues/27
 
### Check List
- [x] Commits are signed per the DCO using --signoff 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: opensearch-maps-server/README.md [hunk 1]
```diff
@@ -1,69 +1,116 @@
...
+{: .note} or build from source code.
+
+
+## Pull Docker image
+
+Open your terminal and run the following command:
+
+`docker pull opensearch/opensearch-maps-server`
+
+## Build docker image from source
```
Comment (heemin32) [inline opensearch-maps-server/README.md:17]: Can we remove this step? Or it shouldn't be next to `Pull Docker image` step but could be better to be inside something like info box?

Comment (heemin32) [reply opensearch-maps-server/README.md:17]: Also, should be Docker with capital D I guess?

File: opensearch-maps-server/README.md [hunk 2]
```diff
@@ -1,69 +1,116 @@
...
 Build docker image under Dockerfile path
 ```
-docker build -t osm-server .
+docker build -t maps-server .
 ```
+---
 
-## Set up server
-We need prepare tiles set before start the server, there are 2 options:
+Note: Below steps use docker image from Docker Hub repository, if you use docker image from code source, change the docker image name correspondingly
```
Comment (heemin32) [inline opensearch-maps-server/README.md:29]:
```suggestion
Note: Below steps use a Docker image from Docker Hub repository, if you use a Docker image built from code, change the Docker image name accordingly
```

File: opensearch-maps-server/README.md (hunk 1 above)
Comment (heemin32) [reply opensearch-maps-server/README.md:17]: This step is not very informative imo.

Comment (junqiu-lei) [reply opensearch-maps-server/README.md:17]: I was thinking to remain this option if someone wants build Docker image from Github source for test. Will make the two options in one step `Get Docker image`

**Review by heemin32 — APPROVED** [review]

**Review by VijayanB — APPROVED** [review]

**Pull request merged**